### PR TITLE
fix: allow ALBEvent type in http-json-body-parser

### PR DESCRIPTION
**Pull request by connorreitz**
### Context

According to the [docs for using middy with an ALBEvent](https://middy.js.org/docs/events/application-load-balancer/), we can use the `httpJsonBodyParserMiddleware` to parse the body of an `ALBEvent`. However, the `ALBEvent` type isn't included in the type definition for the `httpJsonBodyParserMiddleware` function, so if we are using Typescript, the compiler will get mad. This update will allow us to `.use(httpJsonBodyParserMiddleware)` on `ALBEvent` handlers :)

### CODE REVIEW

File: packages/http-json-body-parser/index.d.ts [hunk 1]
```diff
@@ -8,6 +8,6 @@ interface Options {
 
 export type VersionedApiGatewayEvent = APIGatewayEvent | APIGatewayProxyEventV2
```
Comment (willfarrell) [inline packages/http-json-body-parser/index.d.ts:9]: Should `| ALBEvent` go here instead?

**Comment by willfarrell** [comment]
Thanks for the PR! Just a quick question above before I merge it in.

File: packages/http-json-body-parser/index.d.ts (hunk 1 above)
Comment (connorreitz) [reply packages/http-json-body-parser/index.d.ts:9]: I was thinking about that too, the only reason I added it like I did instead is that ALBEvent isn't technically an ApiGatewayEvent. Maybe we could rename `VersionedApiGatewayEvent`:
 `export type RequestEvent = APIGatewayEvent | APIGatewayProxyEventV2 | ALBEvent` 
Or something along those lines. Let me know what you think!
`

Comment (willfarrell) [reply packages/http-json-body-parser/index.d.ts:9]: `RequestEvent ` Sounds good to me

**Pull request merged**